redash/query_runner/clickhouse.py

```diff
@@ -0,0 +1,120 @@
+import json
+import logging
+from redash.query_runner import *
+from redash.utils import JSONEncoder
+logger = logging.getLogger(__name__)
+
+try:
+    import requests
```

Comment:
No need to wrap this in `try`/`except`. `requests` is a direct dependency of the project.